src/style.css

```diff
@@ -1,3 +1,80 @@
+html {
+  --text-color: #000;
+  --hover-color: #00acdc;
+}
+
 body {
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  padding: 0;
   margin: 0;
 }
+
+.nav-item {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: var(--text-color);
+  line-height: 60px;
+}
```

Comment:
Don't use tag names for styling (except html and body)
<img width="818" alt="image" src="https://github.com/user-attachments/assets/1a0839e5-b443-41bb-9d3a-6d45655097c8">